textworld/generator/data/logic/inventory.twl

```diff
@@ -13,6 +15,10 @@ type I {
 
         take/s :: $at(P, r) & $at(s, r) & on(o, s) -> in(o, I);
         put    :: $at(P, r) & $at(s, r) & in(o, I) -> on(o, s);
+
+        examine/I :: in(o, I) -> in(o, I);  # Nothing changes.
```

Comment:
Doesn't work if the inventory is empty?

Reply:
Ah is this for examining an object or the inventory itself?

Reply:
Examining an object that is in the inventory. Taking the inventory (the `inventory` command is handled here https://github.com/Microsoft/TextWorld/pull/81/files#diff-c9ae5bb7789df38acfaf881562a96755R8).